cmd/daemon.go

```diff
@@ -15,14 +21,62 @@ func NewDaemonCmd(global *globalFlags) *cobra.Command {
 		Use:   "daemon",
 		Short: "Run the crypta daemon",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return c.Run(args)
+			if !c.detached {
+				return cmd.Help()
+			}
+
+			return c.run()
+		},
+	}
+
+	start := &cobra.Command{
+		Use:   "start",
+		Short: "Starts the crypta daemon",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.start(args)
+		},
+	}
+
+	stop := &cobra.Command{
+		Use:   "stop",
+		Short: "Stops the crypta daemon",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.stop()
 		},
 	}
 
+	cc.AddCommand(start)
+	cc.AddCommand(stop)
+
+	cc.Flags().BoolVar(&c.detached, "detached", false, "Runs the daemon detached")
+	_ = cc.Flags().MarkHidden("detached")
+
 	return cc
 }
 
-func (c *daemonCmd) Run(args []string) error {
+func (c *daemonCmd) start(args []string) error {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	ex, err := os.Executable()
+	if err != nil {
+		return err
+	}
+
+	// FIXME: Pass global flags to command, in order to be able to set the endpoint
+	args = append(args, "daemon", "--detached")
```

Comment:
Why do you append the `daemon` subcommand to the positional args passed to the `start` command?

Reply:
I just reused the `args` array for passing the new arguments to the new subprocess (as they are empty anyway for the daemon command). This will not be this way in the final PR.